tests/prefetch_related/models.py

```diff
@@ -216,6 +216,16 @@ class Meta:
         ordering = ["id"]
 
 
+class CustomUUIDField(models.UUIDField):
+    def get_prep_value(self, value):
+        return str(value)
+
+
+class ArticleCustomUUID(models.Model):
+    id = CustomUUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
If we were to do the above we could drop the usage of `get_prep_value` entirely.

```suggestion
class ArticleCustomUUID(models.Model):
    id = UUIDField(primary_key=True, default=uuid.uuid4)
```

Reply:
I see. In this case we could keep `CustomUUIDField` as a proper regression test even if `get_prep_value` is not exercised to prevent future breakages by a solution that would. Please ignore my comment.